torch_em/transform/generic.py

```diff
@@ -105,6 +106,52 @@ def __call__(self, inputs):
         return inputs
 
 
+class ResizeLongestSideInputs:
+    def __init__(self, target_shape, target_length=512, is_label=False, is_rgb=False):
```

Comment:
Why do we need the `target_length` here?

Reply:
Yep, you are correct. It's not necessary to have this at all. I'll take care of this.